### ISSUE TITLE
Try He's initialization approach

### ISSUE THREAD

**Issue by marcelsan** [marcelsan/DeepShading#2]
tf.keras.initializers.he_normal(seed=None)

https://arxiv.org/pdf/1502.01852v1.pdf
